config/environments/test.rb

```diff
@@ -10,7 +10,8 @@
   # test suite. You never need to work with it otherwise. Remember that
   # your test database is "scratch space" for the test suite and is wiped
   # and recreated between test runs. Don't rely on the data there!
-  config.cache_classes = true
+  # Disable reloading for integration tests
+  config.cache_classes = ARGV.grep(/test\/integration/).any?
```

Comment:
Shouldn't this be the other way around? Only enable it when it's `none?`?

Also, https://github.com/rails/spring?tab=readme-ov-file#enable-reloading notes it needs to be `false` for spring to work.

Another note: https://github.com/rails/spring?tab=readme-ov-file#temporarily-disabling-spring says:
> If you're using Spring binstubs, but temporarily don't want commands to run through Spring, set the DISABLE_SPRING environment variable.